### ISSUE TITLE
Build is broken

### ISSUE THREAD

**Issue by n1ywb** [webcomponents/shadycss#118]
I get the exact same error when I checkout and try to run locally (latest everything)

https://travis-ci.org/webcomponents/shadycss/builds/254532724

 master 1.0.2

 Commit bb4c0ca
 Compare e92118c..bb4c0ca
 Branch master
Daniel Freedman avatar Daniel Freedman authored and committed

 #321 errored

 Ran for 1 min 42 sec
 7 days ago

```$ gulp test-modules
[17:31:00] Using gulpfile ~/build/webcomponents/shadycss/gulpfile.js
[17:31:00] Starting 'test-modules'...
[17:31:00] Starting 'clean-test-modules'...
[17:31:00] Finished 'clean-test-modules' after 10 ms
[17:31:00] Starting 'test-module-css-parse'...
[17:31:00] Starting 'test-module-custom-style-element'...
[17:31:00] Starting 'test-module-make-element'...
[17:31:00] Starting 'test-module-svg-in-shadow'...
[17:31:00] Starting 'test-module-style-util'...
[17:31:00] Starting 'test-module-style-transformer'...
[17:31:00] Starting 'test-module-style-settings'...
events.js:160
      throw er; // Unhandled 'error' event
      ^
Error: Given moduleName (make-element) is not legal JS identifier. If you need this you can try --extend option
    at error (/home/travis/build/webcomponents/shadycss/node_modules/rollup/dist/rollup.js:177:12)
    at iife (/home/travis/build/webcomponents/shadycss/node_modules/rollup/dist/rollup.js:9029:3)
    at /home/travis/build/webcomponents/shadycss/node_modules/rollup/dist/rollup.js:10200:17
The command "gulp test-modules" failed and exited with 1 during .
```

**Comment by dnlzglr** [comment]
I do not know if this fixes the issue itself, but I was able to get the tests running locally by adding the `extend` flag as proposed by the error message to the `test-module` and `debug` tasks.

In gulpfile.js, line 38ff:
```js
  gulp.task(`test-module-${m}`, () => {
    return rollup({
      entry: `tests/module/${m}.js`,
      format: 'iife',
      moduleName: m,
      extend: true    // <<<< Added flag
    })
    .pipe(source(`${m}.js`, 'tests/module'))
    .pipe(gulp.dest('./tests/module/generated'))
  });
```
and line 83ff:
```js
  gulp.task(`debug-${entry}`, () => {
    return rollup({
      entry: `entrypoints/${entry}.js`,
      format: 'iife',
      moduleName: entry,    // <<<< Changed from '${entry}'
      extend: true    // <<<< Added flag
    })
    /* ... */
  });
```

**Comment by azakus** [comment]
Fixed with 930c7526611c9bf245decf6f751f9838d78f37aa
